docs/how-to-guides/data-basics/5-adding-a-delete-button.md

```diff
@@ -43,10 +43,8 @@ function PagesList( { hasResolved, pages } ) {
 						<td>{ decodeEntities( page.title.rendered ) }</td>
 						<td>
 							<div className="form-buttons">
-								<EditPageButton pageId={ page.id } />
-								{/* ↓ This is the only change in the PagesList component */}
```

Comment:
I'd retain this line – otherwise it can be hard to identify what exactly has changed in this snippet.

Reply:
No problem. Added it back in.